### ISSUE TITLE
Spike: Can Tus support stateless file uploads

### ISSUE THREAD

**Issue by cfarmer-fearless** [CDCgov/data-exchange-upload#42]
The purpose of this spike is to see how Tus manages file upload state when uploading to cloud storage vs local file storage.  This will determine the statefulness of the tusd service.  If we find that the tusd service is stateless, meaning that it uses external resources to track the file upload states, we can safely assume that we can scale this service horizontally.  Otherwise, we need to see how the tusd service manages the file upload state so we can determine what is even possible for scaling.

D.o.D. Diagram and/or report detailing how TUS handles the file upload state on the client and server.  Ultimately answering the question: do tus clients need sticky sessions or not?

**Comment by rohitpanwar** [comment]
https://cdc.sharepoint.com/:w:/r/teams/CDC-Data-Exchange/Shared%20Documents/Operations/Tus%20sticky%20sessions%20spike.docx?d=w74c16452b79e42f9bfc8dfc17591e5ab&csf=1&web=1&e=xwAPVL

**Comment by rmharrison** [comment]
Accepted
TL;DR: Tus does NOT need sticky sessions because state is stored in the managed Azure Blob service.
